controller/volume_controller.go

```diff
@@ -1636,8 +1645,9 @@ func (vc *VolumeController) checkForAutoAttachment(v *longhorn.Volume, e *longho
 	// Do auto attachment for:
 	//   1. restoring/DR volumes.
 	//   2. offline expansion.
+	//	 3. Eviction requested on this volume.
```

Comment:
Wrong indent.

Reply:
I also noticed this, but in my local environment, the numbers are aligned. Let me double check on this.

Reply:
It's normally tab vs space.